pkg/workflows/wasm/host/wasm_test.go

```diff
@@ -168,6 +183,158 @@ func Test_GetWorkflowSpec_Logs(t *testing.T) {
 	}
 }
 
+func Test_Compute_Fetch(t *testing.T) {
+	binary := createTestBinary(fetchBinaryCmd, fetchBinaryLocation, true, t)
+
+	t.Run("OK_default_runtime_cfg", func(t *testing.T) {
+		logger, logs := logger.TestObserved(t, zapcore.InfoLevel)
+		expected := "Valid fetch response"
```

Comment:
@agparadiso This approach works for sure, but you run the risk of having brittle tests if you're checking logs to verify that something is happening or not.

As an alternative, you could transmit the response back to the host, and check against that in the host.

So in other words, your workflow would:

- Compute: make the request, return the response
- Then you get that response back on the host side from the call to Run
- if that response == expected you know the test has worked.